common/test/common/HtmlCleanerTest.scala

```diff
@@ -0,0 +1,129 @@
+package common
+
+import org.jsoup.Jsoup
+import org.scalatest.{FlatSpec, Matchers}
+
+import pagepresser.{InteractiveImmersiveHtmlCleaner}
+
+class HtmlCleanerTest extends FlatSpec with Matchers {
+
+  "InteractiveImmersiveHtmlCleaner" should "only clean interactive immersives" in {
+    val doc = Jsoup.parse("""<html>
+              |<head>
+              |<link rel="canonical" href="https://www.theguardian.com/world/ng-interactive/2020/apr/08/coronavirus-100">
+              |</head>
+              |<body class="is-immersive is-immersive-interactive"></body>
+              |</html>""")
+
+    InteractiveImmersiveHtmlCleaner.canClean(doc) should be(true)
+  }
+
+ "Cleaned interactive immersives" should "have top banner ads removed" in {
+    val doc = Jsoup.parse("""<html>
+              |<body>
+              |<div id="bannerandheader">
+              |<div class="top-banner-ad-container js-top-banner"></div>
+              |<header></header>
+              |</div>
+              |</body>
+              |</html>""".stripMargin)
```

Comment:
I used a multiline string like this so I could more easily create the doc structure - if there's a better way to handle multiline strings please let me know! I used `stripMargin` to ensure the parsing/unparsing resulted in an expected string.